tests/unit/cli/search/test_reindex.py

```diff
@@ -84,6 +84,12 @@ def update_aliases(self, body):
                 else:
                     raise ValueError("Unknown action: {!r}.".format(action))
 
+    def put_settings(self, index, body):
+        pass
```

Comment:
This, and the ``forcemerge`` should be testable somehow to ensure they've been used. Might just be adding a settings attribute to ``FakeESIndices`` that records what the current settings are or something like that.